### DISCUSSION TITLE
AOT Module in Python

### DISCUSSION THREAD

**Discussion by kurt-rhee** [taichi-dev/taichi#8354] [Q&A] [answered]
Hello, 

Is it possible to run the ahead of time compiled kernel in my python script instead of the JIT compiled one?

**Comment by chenzhekl** [comment, marked as answer]
I guess you may be looking for offline cache? https://docs.taichi-lang.org/docs/performance#offline-cache

**Reply by kurt-rhee** [reply]
This is exactly what I was looking for thank you!
